### PR TITLE
Add menubar for translation

### PR DESCRIPTION
**Pull request by Csoyee**
* Add menubar for translation (in ContentView)

* Modify listview layout slightly

### CODE REVIEW

File: Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java [hunk 1]
```diff
@@ -137,4 +137,4 @@ public void insertScriptedDataWithCheckDuplication(String url, String title, Str
     }
     dataBase.close();
   }
-}
+}
```
Comment (nuinuri) [inline Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java]: May need return here

File: Roys/app/src/main/java/com/dwg_karrier/roys/ScriptedURL.java [hunk 2]
```diff
@@ -54,6 +56,20 @@
     this.crawler = new Crawler(url);
   }
 
+  ScriptedURL(String url, int isRead, String title, String content, String repImageUrl, int wordCount) {
+    this.url = url;
+    if(isRead == 1){
+      this.isRead = true;
+    } else {
+      this.isRead = false;
+    }
```
Comment (nuinuri) [inline Roys/app/src/main/java/com/dwg_karrier/roys/ScriptedURL.java]: this.isRead  = isRead == 1

File: Roys/app/src/main/res/layout/contentfragment.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
                 xmlns:app="http://schemas.android.com/apk/res-auto"
-                android:orientation="vertical"
+                xmlns:tools="http://schemas.android.com/tools" android:orientation="vertical"
```
Comment (nuinuri) [inline Roys/app/src/main/res/layout/contentfragment.xml]: break the line?

File: Roys/app/src/main/java/com/dwg_karrier/roys/ContentSwipe.java [hunk 4]
```diff
@@ -28,7 +29,7 @@
   Date finTime;
   Date curTime;
   public int totalPageNum ;
-  private ArrayList<ScriptedURL> unreadPageList;
+  private static ArrayList<ScriptedURL> unreadPageList;
```
Comment (nuinuri) [inline Roys/app/src/main/java/com/dwg_karrier/roys/ContentSwipe.java]: Why static?

Comment (Csoyee) [inline Roys/app/src/main/java/com/dwg_karrier/roys/ContentSwipe.java]: told you directly. 
Made TODO as you say ~

**Comment by Csoyee** [comment]
Apply comment and complete debug.

**Pull request merged**